core/templates/dev/head/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -1,5 +1,5 @@
 <style>
-  admin-navbar .nav > li > a:focus {
+  .hidden-icon-favicon,admin-navbar .nav > li > a:focus {
```

Comment:
Please follow existing CSS conventions in the codebase with regards to spaces after commas, spaces before {, etc. (Look at oppia.css for lots of examples.) Also see https://github.com/oppia/oppia/wiki/Coding-style-guide#css

Also why is a "hidden-icon-favicon" class necessary? It seems to me like something generic like "hidden" (to hide an element) should suffice; that might in fact be part of Bootstrap already.

Reply:
Does "dropdown-icon" sound good?

Reply:
See the coding style guide page I linked you to re class names; they should have the correct prefix.

Also, dropdown-icon is a bit generic, especially if it's only used here. Perhaps oppia-admin-page-dropdown-icon?

Reply:
When I looked closely, I realised my code was way far from convention. Thank You, for referring the link.